packages/client/src/components/LocationPicker.tsx

```diff
@@ -49,6 +49,7 @@ interface IBaseProps {
   disabled?: boolean
   onChangeLocation: (locationId: string) => void
   requiredJurisdictionTypes?: string
+  fuzzy?: boolean
```

Comment:
This makes location searches (for offices, hospitals) fuzzy.

Code can be cherry-picked from this commit:
https://github.com/opencrvs/opencrvs-core/commit/7f0f4f261359377a37f1a9e55bf852f70b3bef97